jenkins-agents/jenkins-agent-ansible/Dockerfile

```diff
@@ -22,25 +21,22 @@ ARG PIP_PKGS="\
         ansible==${ANSIBLE_VERSION} \
 "
 ARG YUM_PKGS="\
-        rh-python36-python-pip \
+        python38-pip \
 "
 
-ADD ubi.repo /etc/yum.repos.d/ubi.repo
-ADD scl_enable /usr/share/container-scripts/
+ADD ubi8.repo /tmp/ubi8.repo
 
-USER root
 RUN set -x \
-     && yum install ${YUM_FLAGS} ${YUM_PKGS} \
-     && yum ${YUM_FLAGS} clean all \
-     && rm -rf /var/cache/yum \
-     && source scl_source enable rh-python36 \
+     && rm -f /etc/yum.repos.d/*.repo \
+     && mv /tmp/ubi8.repo /etc/yum.repos.d/ \
+     && dnf install ${YUM_FLAGS} ${YUM_PKGS} \
+     && dnf ${YUM_FLAGS} clean all \
+     && rm -rf /var/cache/dnf \
+     && alternatives --set python3 /usr/bin/python3.8 \
      && python3 -m pip install ${PIP_PKGS} \
     && echo
 
 USER 1001
 ENV \
     LANG=en_US.UTF-8 \
     LC_ALL=en_US.UTF-8 \
-    ENV=/usr/share/container-scripts/scl_enable \
-    BASH_ENV=/usr/share/container-scripts/scl_enable \
-    PROMPT_COMMAND="./usr/share/container-scripts/scl_enable"
```

Comment:
since scl_enable is removed in the setup, is it still needed in git? cant see it being deleted.

Reply:
Good find, I've removed them + a few other files/dirs that are not used in the images.